react/ProductVariations.tsx

```diff
@@ -10,7 +10,10 @@ const ProductVariations: FunctionComponent = () => {
     <div className={`c-muted-1 f6 lh-copy ${opaque(item.availability)}`}>
       {item.skuSpecifications.map((spec: SKUSpecification, idx: number) => {
         return (
-          <div id={`${idx}`} key={idx}>
+          <div
+            id={`${idx}-${spec.fieldName}-specification`}
```

Comment:
`idx` is not an identifier, it's the index in the `skuSpecifications` array. For that reason the `id` you picked can still cause collisions. It's better to use `item.id` and concatenate it with the `spec.fieldName`.

Reply:
Gotcha!